runtime/vm/createramclass.cpp

```diff
@@ -197,6 +197,18 @@ static void initializeClassLinks(J9Class *ramClass, J9Class *superclass, J9Memor
 #define MAGIC_ACCESSOR_IMPL "jdk/internal/reflect/MagicAccessorImpl"
 #endif /* JAVA_SPEC_VERSION == 8 */
 
+static inline J9Class *
+getArrayClass(J9Class *elementClass, UDATA options) {
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+	if (J9_ARE_ALL_BITS_SET(options, J9_FINDCLASS_FLAG_CLASS_OPTION_NULL_RESTRICTED_ARRAY)) {
+		return elementClass->nullRestrictedArrayClass;
+	} else
+#endif /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
+	{
+		return elementClass->arrayClass;
+	}
+}
+
```

Comment:
This would be better if it was reformatted like:
```
static inline J9Class *
getArrayClass(J9Class *elementClass, UDATA options)
{
#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
        if (J9_ARE_ALL_BITS_SET(options, J9_FINDCLASS_FLAG_CLASS_OPTION_NULL_RESTRICTED_ARRAY)) {
                return elementClass->nullRestrictedArrayClass;
        }
#endif /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
        return elementCalss->arrayClass;
}
```

Reply:
I think the `else` makes it more readable personally.

Reply:
The `else` is just redundant if the body of the `if` has a `return`, and leads to funky formatting with the if-directives. Just my preference.